images/macos/helpers/Common.Helpers.psm1

```diff
@@ -156,3 +156,21 @@ function Add-EnvironmentVariable {
 function isVeertu {
     return (Test-Path -Path "/Library/Application Support/Veertu")
 }
+
+Function Test-CommandExists {
+ param
+ (
+     $command
+ )
+
+ $oldPreference = $ErrorActionPreference
+ $ErrorActionPreference = 'Stop'
+
+ try {
+     if(Get-Command $command){ $true }
```

Comment:
Can't we use `-ErrorAction SilentlyContinue` and leave `$ErrorActionPreference` variable alone?

Reply:
well, I have no particular opinion on that, I'll use your suggestion after rebase